pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -138,6 +139,16 @@ func (w *watcher) run(ctx context.Context, repo git.Repo, repoCfg config.PipedRe
 					zap.String("branch", repo.GetClonedBranch()),
 					zap.Error(err),
 				)
+				if err := repo.Clean(); err != nil {
+					w.logger.Error("failed to remove repo directory",
+						zap.String("path", repo.GetPath()),
+						zap.Error(err),
+					)
+				}
+				w.logger.Info("Try to re-clone because it's more likely to be unable to pull the next time too",
+					zap.String("repo-id", repoCfg.RepoID),
+				)
+				repo, _ = w.cloneRepo(ctx, repoCfg)
```

Comment:
Is it ok to ignore the retuned error?

Reply:
All we have to do here is just emit error log. `cloneRepo()` does it, that's why it ignores the error.

Reply:
But conventionally, cloneRepo() should just give back an error and caller emits log.

Reply:
👍